src/librustc_session/config.rs

```diff
@@ -771,7 +774,7 @@ macro_rules! options {
                                                                value, $outputname,
                                                                key, type_desc))
                         }
-                        (None, None) => bug!()
+                        (None, None) => panic!()
```

Comment:
nit: I think the `unreachable!` macro is a better fit

Reply:
Hm, I am less sure that this place is unreachable -- I prefer to avoid that variant of panic! unless I feel somewhat confident that replacing it with e.g. std::hint::unreachable_unchecked() would be safe, even if that isn't something that I'd ever do without benchmarks etc.

I've not read this code closely enough to figure out if that's the case here.

Reply:
Ok